lib/morse_ruby.rb

```diff
@@ -0,0 +1,19 @@
+def decode(str)
+  morse_hash = { '.-': 'a', '-...': 'b', '-.-.': 'c', '-..': 'd', '.': 'e', '..-.': 'f', '--.': 'g', '....': 'h',
+                 '..': 'i', '.---': 'j', '-.-': 'k', '.-..': 'l', '--': 'm', '-.': 'n', '---': 'o', '.--.': 'p',
+                 '--.-': 'q', '.-.': 'r', '...': 's', '-': 't', '..-': 'u', '...-': 'v', '.--': 'w', '-..-': 'x',
+                 '-.--': 'y', '--..': 'z' }
+  delimiters = ['  ']
+  words = str.split(Regexp.union(delimiters))
+
+  result = ''
+  words.each do |word|
+    word = word.split
+    word.each { |symbol| result += morse_hash[:"#{symbol}"] }
+    result += ' '
+  end
+
+  puts(result)
+end
+
```

Comment:
- [ ] Kindly ensure you have all three methods for decoding character, a word and the whole message , currently you have only one, its a project requirement:
![image](https://user-images.githubusercontent.com/61408860/178763443-7fc181e7-9cd8-4ca3-bff0-f0ab4f52df8f.png)